Ustring.php

```diff
@@ -220,6 +212,26 @@ public function __construct($string = null)
     }
 
     /**
+     * Check if ext/mbstring is available.
+     *
+     * @return  bool
+     */
+    public static function checkMbString()
+    {
+        return function_exists('mb_substr');
```

Comment:
@Hywan my bad, a bit of RTFM on `patchwork/utf8` revealed that `\Patchwork\Utf8\Bootup::initAll();` wil declare the functions.

Good for me then :+1: